operator_ui/src/components/JobRuns/TaskExpansionPanel.tsx

```diff
@@ -22,100 +27,137 @@ const fontStyles = () =>
-        <Item keyOne="Params" valOne={par[0]} keyTwo="Values" valTwo={par[1]} key={idx} />
+      {Object.entries(params || {}).map((p, idx) => (
+        <Item
+          key={idx}
+          colATitle="Params"
+          colAValue={p[0]}
+          colBTitle="Values"
+          colBValue={p[1]}
+        />
       ))}
     </div>
   )
 }
 
-const renderResult = (result: string) => (
-  <Item keyOne="Result" valOne="Task Run Data" keyTwo="Values" valTwo={result} />
-)
+interface IResultProps {
+  run: ITaskRun
+}
+
+const Result = ({ run }: IResultProps) => {
+  const result = run.result && run.result.data && run.result.data.result
+
+  return (
+    <Item
+      colATitle="Result"
+      colAValue="Task Run Data"
+      colBTitle="Values"
+      colBValue={result}
+    />
+  )
+}
 
-const TaskExpansionPanel = ({ children }: { children: IJobRun }) => {
-  const initiator: IInitiator = children.initiator
-  const taskRuns: ITaskRuns = children.taskRuns
+interface IProps {
+  jobRun: IJobRun
+}
+
+const TaskExpansionPanel = ({ jobRun }: IProps) => {
+  const initiator = jobRun.initiator
 
   return (
     <Grid container spacing={0}>
       <Grid item xs={12}>
         <StatusItem
           summary={capitalize(initiator.type)}
-          status={children.status}
+          status={jobRun.status}
           borderTop={false}
           confirmations={0}
           minConfirmations={0}
         >
-          {renderInitiator(initiator.params)}
+          <Initiator params={initiator.params} />
         </StatusItem>
       </Grid>
-      {taskRuns.map(taskRun => (
-        <Grid item xs={12} key={taskRun.id}>
-          <StatusItem
-            summary={capitalize(taskRun.task.type)}
-            status={taskRun.status}
-            confirmations={taskRun.task.confirmations}
-            minConfirmations={taskRun.minimumConfirmations}
-          >
-            <Grid container direction="column">
-              <Grid item>{renderParams(taskRun.task && taskRun.task.params)}</Grid>
-              <Grid item>{renderResult(taskRun.result && taskRun.result.data && taskRun.result.data.result)}</Grid>
-            </Grid>
-          </StatusItem>
-        </Grid>
-      ))}
+      {jobRun.taskRuns.map((taskRun: ITaskRun) => {
+        return (
+          <Grid item xs={12} key={taskRun.id}>
+            <StatusItem
+              borderTop
+              summary={capitalize(taskRun.task.type)}
+              status={taskRun.status}
+              confirmations={taskRun.task.confirmations}
+              minConfirmations={taskRun.minimumConfirmations}
+            >
+              <Grid container direction="column">
+                <Grid item>
+                  {taskRun.task && <Params params={taskRun.task.params} />}
+                </Grid>
+                <Grid item>
+                  <Result run={taskRun} />
+                </Grid>
+              </Grid>
+            </StatusItem>
+          </Grid>
+        )
+      })}
```

Comment:
overall good changes here too, but the changes in this file might deserve a separate PR